app/templating/summary/question.py

```diff
@@ -67,11 +80,11 @@ def _build_checkbox_answers(cls, answer, answer_schema):
         multiple_answers = []
         for option in answer_schema['options']:
             if option['value'] in answer:
-                if option['value'] == 'other':
+                if option['value'].lower() == 'other':
                     summary_option_display_value = cls._get_checkbox_other_display_value(answer, answer_schema, option)
-                    multiple_answers.append(summary_option_display_value)
+                    multiple_answers.append((summary_option_display_value, True))
```

Comment:
We should use a dict here rather than a tuple so we can label the values and replace the references in the templates of `0` and `1` with keys e.g. `multiple_answers.append({'label': option['label'], 'is_other_answer': False})`.  This would make this code and the templates much clearer.